plugin/xcode.vim

```diff
@@ -94,7 +99,13 @@ function! s:open(path)
 endfunction
 
 function! s:switch(target)
-  execute '!sudo xcode-select -s' . s:cli_args(a:target)
+  if has('nvim')
```

Comment:
I think we're talking about different things, but let me know if I'm wrong. 

I meant something like this:

```vim
function! s:runner(){
  if has('nvim')
    return '!'
  else
    return 'terminal'
  end
endfunction

let s:default_runner_command = s:runner() . ' {cmd}'

function! s:switch(target)
  execute s:runner() . ' sudo  xcode-select -s' . s:cli_args(a:target)
endfunction
```

One downside to that code is that (I think) `s:runner` has to be defined before `s:default_runner_command` is defined.